lib/tests/streamlit/pickle_test_removeme.py

```diff
@@ -0,0 +1,76 @@
+import pickle
```

Comment:
I take it from the name this was for verifying things while you worked and not intended to be merged in.